conda/recipes/legate-boost/meta.yaml

```diff
@@ -80,12 +80,12 @@ requirements:
     {% if gpu_enabled_bool %}
     - cuda-version ={{ cuda_version }}
     - cuda-cudart-dev
-    - cunumeric {{ legate_version }}
-    - legate-core {{ legate_version }}
+    - cunumeric {{ legate_version }} =*_gpu
+    - legate {{ legate_version }} =*_gpu
```

Comment:
oh hey great! I'm glad they now added `_gpu` to the build strings for `legate` / `cunumeric`. When I'd asked about that before, the folks maintaining `legate` / `cunumeric` were against it: https://github.com/nv-legate/legate.core.internal/pull/1035#discussion_r1702268130